api/helpers.go

```diff
@@ -91,3 +107,196 @@ func paginationToRequest[T any](rows []T, dbPageSize int32, cursor string, goFor
 	}
 	return rows, nextCursor, prevCursor
 }
+
+// CensusOptions envolves the required parameters to create and publish a
+// census merkle tree
+type CensusOptions struct {
+	ID      uint64
+	Type    models.Census_Type
+	Holders map[common.Address]*big.Int
+}
+
+// CreateAndPublishCensus function creates a new census tree based on the
+// options provided and publishes it to IPFS. It needs to persist it temporaly
+// into a internal trees database. It returns the root of the tree, the IPFS
+// URI and the tree dump.
+func CreateAndPublishCensus(
```

Comment:
Personally I don't like this way to break lines for the function signature... I'd try to split it in two lines of similar size. But that's just a personal preference.

Reply:
me neither but it is generated by `gofumpt` :(